.github/workflows/pubsys-exercise.yaml

```diff
@@ -0,0 +1,27 @@
+name: pubsys
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - 'tools/**'
+jobs:
+  check:
+    runs-on: [self-hosted, linux, x64]
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup toolchain install 1.53.0 && rustup default 1.53.0
+      - run: cargo install --version 0.30.0 cargo-make
+      - run: cargo make -e BUILDSYS_JOBS=12
```

Comment:
Why do we override JOBS here but not in the build workflow?

Reply:
We do override it in the build workflow
https://github.com/bottlerocket-os/bottlerocket/blob/e4e610e5077a712e143cdb667e7f3a5447aa5364/.github/workflows/build.yml#L35

Reply:
Oh, sorry, I guess I got mixed up with the unit-tests line.